src/ai_client.c

```diff
@@ -3,6 +3,8 @@
  *
  * Copyright (C) 2009-2011 Keldon Jones
  *
+ * Source file modified by J.-R. Reinhard, October 2016.
```

Comment:
Faking the version is a big hack. I document it in client.c, we should strive to remove this as soon as it is not required anymore. 

This stems from testing for too new clients. Is such a test necessary? Should  we remove it from the code?